### PR TITLE
Fix visualization of blocked_note and is_blokked fields

### PR DESCRIPTION
**Pull request by bameda**
Merged with:
 - https://github.com/taigaio/taiga-contrib-slack/pull/8
 - https://github.com/taigaio/taiga-contrib-hall/pull/3
 - https://github.com/taigaio/taiga-contrib-hipchat/pull/6

### CODE REVIEW

File: taiga/projects/history/templates/emails/includes/fields_diff-html.jinja [hunk 1]
```diff
@@ -121,7 +123,15 @@
...
+                </td>
+            </tr>
+    {# CONTENT #}
+        {% elif field_name in ["blocked_note_diff"] %}
+            <tr>
+                <td colspan="2">
+                    <h3>{{ verbose_name(obj_class, field_name) }}</h3>
                     <p>{{ mdrender(project, values.1) }}</p>
                 </td>
             </tr>
```
Comment (superalex) [inline taiga/projects/history/templates/emails/includes/fields_diff-html.jinja]: why not mixing the two eilf's for description_diff and content_diff?

{% elif field_name in ["description_diff", "content_diff"] %}

Comment (bameda) [inline taiga/projects/history/templates/emails/includes/fields_diff-html.jinja]: You are right, description, content and blocked_note are equals

**Comment by superalex** [comment]
In the text version of the email I can't see the "blocked note" if I block something (it's appearing ok in the html email)

**Comment by superalex** [comment]
Great! :+1:

**Pull request merged**